### PR TITLE
VY: Add Autologout functionality

### PR DESCRIPTION
**Pull request by jediandrey**
1) enable 'Open Y Virtual YMCA Autologout' module
2) go to /admin/virtual-y/openy-gc-autologout and setup some little value e.g. 10 seconds
3) login into Virtual Y by any user (except admin)
4) ensure you've been logged out after 10 seconds of INACTIVITY

### CODE REVIEW

**Comment by jediandrey** [comment]
retest

File: modules/openy_gc_autologout/openy_gc_autologout.module [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }
+}
+
+/**
+ * Check if provided paragraph exists on the node.
+ */
+function openy_gc_autologout_check_if_prgf_at_node(NodeInterface $node, $paragraph_id) {
+  $connection = Database::getConnection();
+
+  $result = $connection->select('paragraphs_item_field_data', 'pd')
```
Comment (anpolimus) [inline modules/openy_gc_autologout/openy_gc_autologout.module:42]: I think it is good chance to make a separate service for this code and for existed one:
https://github.com/fivejars/openy_gated_content/blob/master/modules/openy_gc_auth/src/EventSubscriber/VirtualYLoginRedirect.php#L155

**Review by anpolimus — CHANGES_REQUESTED** [review]

File: modules/openy_gc_autologout/openy_gc_autologout.module [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+/**
+ * Auto logout users.
+ */
+function openy_gc_autologout_users() {
+  $results = \Drupal::database()->select('sessions', 's')
+    ->fields('s', ['uid', 'sid ', 'timestamp'])
+    ->execute()
+    ->fetchAll();
```
Comment (hamrant) [inline modules/openy_gc_autologout/openy_gc_autologout.module:66]: Let's add some limit here

**Review by hamrant — CHANGES_REQUESTED** [review]

**Review by anpolimus — APPROVED** [review]

**Comment by anpolimus** [comment]
Code is ok.

**Review by hamrant — APPROVED** [review]

**Comment by NadyaRudenko** [comment]
@jediandrey , tested on Safari. 
I was not logged out
When I set a new time at /admin/virtual-y/openy-gc-autologout and click 'Save configuration', I can see the default time again. I'm not sure if this is right
image

**Comment by jediandrey** [comment]
@NadyaRudenko fixed, please try again

**Comment by NadyaRudenko** [comment]
Retested, fixed. Thank you!

**Review by hamrant — APPROVED** [review]

**Pull request merged**